pkg/controllers/managementuser/secret/secret.go

```diff
@@ -102,6 +102,10 @@ func (n *NamespaceController) sync(key string, obj *corev1.Namespace) (runtime.O
 	if obj.Annotations[projectIDLabel] != "" {
 		parts := strings.Split(obj.Annotations[projectIDLabel], ":")
 		if len(parts) == 2 {
+			if parts[1] == "" {
+				logrus.Tracef("obj.Annotations[projectIDLabel] has an empty project name for cluster: %s", parts[0])
```

Comment:
This might be better suited to debug, and prepend the message with something to help say where this is coming from i.e. `[NamespaceController]`

But, if you are deleting this annotation anyways I wouldn't expect to hit this after the first start of this version.

Reply:
I agree. I'll change it to a debug statement and make the message more clear